.github/workflows/ci.yml

```
@@ -0,0 +1,167 @@
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Build
+        run: pnpm build
+
+      - name: Upload build artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist
+          path: dist/
+          retention-days: 7
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)